grazer/src/main/java/edu/asu/diging/grazer/core/conceptpower/impl/ConceptpowerCache.java

```diff
@@ -0,0 +1,39 @@
+package edu.asu.diging.grazer.core.conceptpower.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.grazer.core.conceptpower.IASyncConceptUpdater;
+import edu.asu.diging.grazer.core.conceptpower.IConceptpowerCache;
+import edu.asu.diging.grazer.core.conceptpower.IConceptpowerConnector;
+import edu.asu.diging.grazer.core.conceptpower.db.IConceptDatabaseConnection;
+import edu.asu.diging.grazer.core.model.IConcept;
+
+@Service
+public class ConceptpowerCache implements IConceptpowerCache {
+	
+    @Autowired
+    private IConceptDatabaseConnection conceptDB;
+	
+    @Autowired
+    private IConceptpowerConnector connector;
+	
+    @Autowired
+    private IASyncConceptUpdater conceptUpdater;
+	
+    @Override
+    public IConcept getConceptByUri(String id) {
```

Comment:
why is the method called "getConceptByUri" but the parameter is called "id"? which one is it?